lib/deck.rb

```diff
@@ -0,0 +1,21 @@
+require 'pry'
+class Deck
+  attr_reader :cards, :count, :cards_in_category
```

Comment:
Don't need a `:cards_in_category` in the attr_reader, since it is already a method.